tests/tests.py

```diff
@@ -92,19 +103,6 @@ def test_sumif(self):
         self.assertTrue(results['count'] == q.count())
         self.assertEqual(results['rows'], list(q))
 
-    def test_cannot_start_from_forbidden_model(self):
```

Comment:
Why are we deleting this one?

Reply:
I think we've moved to being able to interrogate from any allowed model, but I'll double check this

Reply:
As discussed, I'll write a new test to confirm that  you can't start from a forbidden model

Reply:
Test has been written